cwms_radar_api/src/main/java/cwms/radar/data/dao/OfficeDao.java

```diff
@@ -74,7 +76,7 @@ public Optional<Office> getOfficeById(String officeId) {
                 view.OFFICE_TYPE.as("type"),
                 view.REPORT_TO_OFFICE_ID.as("reportsTo"))
             .from(view)
-            .where(view.OFFICE_ID.eq(officeId.toUpperCase()))
+            .where(view.OFFICE_ID.eq(upper(officeId)))
```

Comment:
should also have view.OFFICE_ID.upper or whatever it actually is. (On the off chance we ever have a non-upper case office ID).